actions/update-link-index/README.md

```diff
@@ -0,0 +1,63 @@
+<!-- 
+    this documentation was generated by https://github.com/reakaleek/gh-action-readme 
+    with the command `VERSION=main gh action-readme update`    
+-->
+
+# <!--name-->Update Link Index<!--/name-->
+
+<!--description-->
+This action updates the link index for the given link reference file.
+<!--/description-->
+
+## Inputs
+
+<!--inputs-->
+| Name                  | Description                                                    | Required | Default                           |
+|-----------------------|----------------------------------------------------------------|----------|-----------------------------------|
+| `link_reference_file` | The path to the link reference file                            | `false`  | `.artifacts/docs/html/links.json` |
+| `aws_account_id`      | The AWS account ID to generate the role ARN for                | `false`  | `197730964718`                    |
+| `aws_region`          | The AWS region to use                                          | `false`  | `us-east-1`                       |
+| `aws_s3_bucket_name`  | The name of the S3 bucket to upload the link reference file to | `false`  | `elastic-docs-link-index`         |
+<!--/inputs-->
+
+## Outputs
+<!--outputs-->
+| Name | Description |
+|------|-------------|
+<!--/outputs-->
+
+## Usage
+
+<!--usage action="elastic/docs-builder/actions/update-link-index" version="env:VERSION"-->
+```yaml
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*.*.*"
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      pages: write
+      id-token: write
+    environment:
+      name: github-pages
+      url: ${{steps.deployment.outputs.page_url}}
+    steps:
+     - name: Publish Github
```

Comment:
spacing here looks off I think this block needs another space?

Reply:
Thank you!

https://github.com/elastic/docs-builder/pull/207/commits/8efbd7fd951501698e6308265ecb6a45c95661ae